src/FixedDataTableRow.js

```diff
@@ -30,7 +30,7 @@ var HEADER_BORDER_BOTTOM_WIDTH = 1;
  * This component should not be used directly by developer. Instead,
  * only <FixedDataTable /> should use the component internally.
  */
-class FixedDataTableRowImpl extends React.Component {
+class FixedDataTableRow extends React.Component {
   /**
```

Comment:
We will pass all the props from FixedDataTable to FixedDataTableRow .So, I changed the name of FixedDataTableRowImpl to FixedDataTableRow